animation_nodes/nodes/gpencil/set_gp_layer_attributes.py

```diff
@@ -0,0 +1,78 @@
+import bpy
+from bpy.props import *
+from ... events import propertyChanged
+from ... data_structures import GPLayer, VirtualPyList, VirtualDoubleList, VirtualLongList
+from ... base_types import AnimationNode, VectorizedSocket
+
+class SetGPLayerAttributesNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_SetGPLayerAttributesNode"
+    bl_label = "Set GP Layer Attributes"
+    errorHandlingType = "EXCEPTION"
+
+    useLayerList: VectorizedSocket.newProperty()
+    useNameList: VectorizedSocket.newProperty()
+    useBlendModeList: VectorizedSocket.newProperty()
+    useOpacityList: VectorizedSocket.newProperty()
+    usePassIndexList: VectorizedSocket.newProperty()
+
+    def create(self):
+        self.newInput(VectorizedSocket("GPLayer", "useLayerList",
+            ("Layer", "layers"), ("Layers", "layers")), dataIsModified = True)
+        self.newInput(VectorizedSocket("Text", "useNameList",
+            ("Name", "layerNames"), ("Names", "layerNames")), value = 'AN-Layer')
+        self.newInput(VectorizedSocket("Text", "useBlendModeList",
+            ("Blend Mode", "blendModes"), ("Blend Modes", "blendModes")), value = 'REGULAR', hide = True)
+        self.newInput(VectorizedSocket("Float", "useOpacityList",
+            ("Opacity", "opacities"), ("Opacities", "opacities")), value = 1, hide = True)
+        self.newInput(VectorizedSocket("Integer", "usePassIndexList",
+            ("Pass Index", "passIndices"), ("Pass Indices", "passIndices")), value = 0, hide = True)
+        self.newOutput(VectorizedSocket("GPLayer", ["useLayerList", "useNameList", "useBlendModeList",
```

Comment:
Put a new line before `self.newOutput`. Try to organize this a bit more, maybe:
```python
self.newOutput(VectorizedSocket("GPLayer",
    ["useLayerList", "useNameList", "useBlendModeList", "useOpacityList", "usePassIndexList"],
    ("Layer", "outLayer"), ("Layers", "outLayers")))
```

Reply:
Done.